apps/src/javalab/JavalabView.jsx

```diff
@@ -151,18 +186,12 @@ class JavalabView extends React.Component {
                 </button>
               </div>
               <div style={style.buttons}>
+                <JavalabSettings style={this.getButtonStyles(true)}>
```

Comment:
NIT: Since `true` is a primitive, it's difficult to understand what it represents. I've seen people mitigate this in the past by adding a comment i.e. `this.getButtonStyles(true /* isSettingsButton */)`